index.html

```diff
@@ -0,0 +1,103 @@
+    <p>Chrome, Firefox, and Edge are the most used browsers today.</p>
+    <section id= "Javascript" class="main-section">
+      <header>Javascript</header>
+      <p>JavaScript is the Programming Language for the Web.</p>
+      <p>JavaScript can calculate, manipulate and validate data.</p>
+      <code>
+        function colorChangeToBlue(){
+          getElementById("element").style.color="blue";
+        }
+      </code> 
+      <code>
+        function add(a,b){
+          return a+b;
+        }
+      </code> 
+      <code>
+        function ChangeFontSize(){
+          getElementById("element").style.FontSize=200px;
+        }
+      </code> 
+      <code>
+        function multiplication(a,b){
+          retunr a*b;
+        }
+      </code> 
+      <code>
+        function GreaterThan(a,b){
+          return a > b ? "A is bigger":"B is bigger";
+        }
+      </code>
+      <p>The documentation was search in part from= <a href="https://www.w3schools.com/whatis/whatis_js.asp">W3schools.com</a></p>
+    </section>
+  
+    <section id= "Html" class="main-section">
+      <header>HTML</header>
+      <p>HTML stands for HyperText Markup Language.</p>
+      <p> It is a standard markup language for web page creation. It allows the creation and structure of sections, paragraphs, and links using HTML elements (the building blocks of a web page) such as tags and attributes. </p>
+      
+    </section>
+  
+    <section id= "Css" class="main-section">
+      <header>CSS</header>
+      <p>CSS is the language we use to style a Web page.</p>
+      <p>Here are some facts about CSS:</p>
+      
+      <ul>
+      <li>CSS stands for Cascading Style Sheets</li>
+      <li>CSS describes how HTML elements are to be displayed on screen, paper, or in other media</li>
+      <li>CSS saves a lot of work. It can control the layout of multiple web pages all at once</li>
+      <li>External stylesheets are stored in CSS files</li>
+      <li>Tools like "bootstrap" can save time</li>
+      </ul>
+    </section>
+  
+    <section id= "Git_and_Github" class="main-section">
+      <header>Git and Github</header>
+      <p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.</p>
+      <p>GitHub is a for-profit company offering a cloud-based Git repository that helps developers store, manage, track and control changes to their code.</p>
+      
+    </section>
+  
+    <section id= "Soft_skills" class="main-section">
+      <header>Soft skills</header>
+      <p>Soft skills are the skills that enable you to fit in at a workplace. </p>
+      <p>hey include your personality, attitude, flexibility, motivation, and manners.</p>
+       
```

Comment:
Be careful with blank lines and indentation.
- [ ] Do not add blank lines, spaces, or indentations without a reason.

Reply:
- [ ] Remember to always be polites and make suggestions rather than just telling what to do
for example: This is a polite suggestion: *"I think it would be better to not add blank lines and spaces without any particular reason, as it will help maintain a readable style in your codebase. Extra blank lines take up extra space, using a linter will automatically fix this issue for us"*

Reply:
it would be better to not add blank lines and spaces without any particular reason, as it will help maintain a readable style in your codebase. Extra blank lines take up extra space, using a linter will automatically fix this issue for us.

- [ ] Kindly delete the extra blank lines in order to make your code readable
- [ ] kindly add linters by catching syntax errors more efficiently. In this link you can see how to do that [https://github.com/microverseinc/linters-config](https://github.com/sanieni6/code-review/pull/1/url)